packages/abi-gen/CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # CHANGELOG
 
+## v0.2.3 - _TBD, 2018_
+
+    * Add a `backend` parameter that allows you to specify your backend (web3 or ethers). Ethers auto-converts small ints to numbers (#413)
```

Comment:
* Add a `backend` parameter that allows you to specify the Ethereum library you use in your application ([Web3.js](https://github.com/ethereum/web3.js) or [Ethers.js](https://github.com/ethers-io/ethers.js)). Ethers auto-converts small ints to numbers whereas Web3 doesn't. Defaults to `Web3`.